store/tikv/scan.go

```diff
@@ -168,6 +168,12 @@ func (s *Scanner) getData(bo *Backoffer) error {
 			return errors.Trace(errBodyMissing)
 		}
 
+		checkerr := s.snapshot.store.CheckVisibility(s.startTS())
+
```

Comment:
useless empty line

Reply:
OK, I have delete the empty line. Thanks~